FrontEnd/index.html

```diff
@@ -35,53 +36,7 @@ <h2>Designer d'espace</h2>
 		</article>
 	</section>
 	<section id="portfolio">
-		<h2>Mes Projets</h2>
-		<div class="gallery">
-			<figure>
```

Comment:
Garder la gallery "hardcode" pour avoir une gallery même si le BE ne fonctionne pas. Avec JS, lorsqu'on recupère les data depuis BE, supprimer la gallery du DOM et la remplacer par la data reçu par BE